src/gui/presets.cpp

```diff
@@ -463,101 +463,101 @@ void FurnaceGUI::initSystemPresets() {
   ));
   cat.systems.push_back(FurnaceGUISysDef(
     "PC + Covox Sound Master", {
-      DIV_SYSTEM_AY8930, 64, 0, 3,
```

Comment:
Hmmm... the PC Speaker is last because it's not that important compared to the expansion chip.

Reply:
Ah, I'll revert PC speaker presets then